__test__/main.test.ts

```diff
@@ -0,0 +1,70 @@
+import * as core from "@actions/core";
+import * as exec from "@actions/exec";
+
+import * as linux from "../src/setup-gazebo-linux";
+import * as utils from "../src/utils";
+
+describe("workflow test without input", () => {
+	beforeAll(() => {
+		jest.spyOn(exec, "exec").mockImplementation(jest.fn());
+	});
+
+	afterAll(() => {
+		jest.resetAllMocks();
+	});
+
+	it("run Linux workflow without", async () => {
+		await expect(linux.runLinux()).rejects.toThrow();
+	});
+});
+
+describe("workflow test with a invalid distro input", () => {
+	beforeAll(() => {
+		jest.spyOn(exec, "exec").mockImplementation(jest.fn());
+		jest.spyOn(core, "getInput").mockReturnValue("dome");
+	});
+
+	afterAll(() => {
+		jest.resetAllMocks();
+	});
+
+	it("run Linux workflow without", async () => {
+		await expect(linux.runLinux()).rejects.toThrow();
+	});
+});
+
+describe("workflow test with a valid distro input", () => {
+	beforeAll(() => {
+		jest.spyOn(exec, "exec").mockImplementation(jest.fn());
+		jest.spyOn(core, "getInput").mockReturnValue("citadel");
```

Comment:
```suggestion
		jest.spyOn(core, "getInput").mockReturnValue("harmonic");
```
citadel will stop to be supported soon, let's use harmonic.